src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests/ProcessTests.cs

```diff
@@ -370,11 +368,16 @@ public void TestProcessorTime()
         [Fact]
         public void TestProcessStartTime()
         {
+            DateTime timeBeforeCreatingProcess = DateTime.UtcNow;
             Process p = CreateProcessInfinite();
             try
             {
                 p.Start();
-                Assert.True(p.StartTime.ToUniversalTime() <= DateTime.UtcNow, string.Format("Process StartTime is larger than later time."));
+
+                // Ensure process has been started, should not throw.
+                Process accessProcessFromMachine = Process.GetProcessById(p.Id);
```

Comment:
Nope, i am just checking if the process with p.id is started on the machine, if not accessing p.id will throw. I can remove the unnecessary allocation to accessProcessFromMachine.